### PR TITLE
[chore] Add ignore for package com.github.johnrengelman.shadow

### PR DESCRIPTION
**Pull request by vasireddy99**
**Description:** 
Add ignore for package com.github.johnrengelman.shadow during the dependabot run to avoid major version updates.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -71,7 +71,9 @@ updates:
         update-types:
           - "version-update:semver-major"
           - "version-update:semver-minor"
-
+      - dependency-name: "com.github.johnrengelman.shadow"
+          update-types:
```
Comment (bryan-aguilar) [inline .github/dependabot.yml]: The name is a bit confusing but `update-types` is a list of update types to ignore. So we would want `version-update:semver-major` here. See dependabot docs [here](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#specifying-dependencies-and-versions-to-ignore).

Comment (bryan-aguilar) [reply .github/dependabot.yml]: Also, this tabbing does not look correct.

File: .github/dependabot.yml [hunk 2]
```diff
@@ -71,7 +71,9 @@ updates:
...
+          - "version-update:semver-major"
```
Comment (bryan-aguilar) [inline .github/dependabot.yml:76]: does this need to be indented or is it already? I can't tell

Comment (vasireddy99) [reply .github/dependabot.yml:76]: Oh yes yes, it is indented in my IDE

image

Comment (vasireddy99) [reply .github/dependabot.yml:76]: Oh yes yes, it is indented in my IDE. double checked and recommited

image

**Review by bryan-aguilar — APPROVED** [review]

**Pull request merged**